docs/test/accessibility.md

```diff
@@ -0,0 +1,20 @@
+# Accessibility
```

Comment:
I would merge all the "test" pages into one single "test.md" pages under the accessibility developer docs.